system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -169,6 +169,72 @@
                    :Term "ENVIRONMENTAL IMPACTS"
                    :Topic "HUMAN DIMENSIONS"}]))))))
 
+(deftest bulk-update-science-keywords-2
+  ;; This test is the same as the previous bulk-update-science-keywords test except
+  ;; that it shows that update-value could be an array of objects, as well as object, when update-type is "ADD_TO_EXISTING" 
+  ;; Ingest a collection in each format with science keywords to update
+  (let [concept-ids (ingest-collection-in-each-format science-keywords-umm)
+        _ (index/wait-until-indexed)
+        bulk-update-body {:concept-ids concept-ids
+                          :name "TEST NAME"
+                          :update-type "ADD_TO_EXISTING"
+                          :update-field "SCIENCE_KEYWORDS"
+                          :update-value [{:Category "EARTH SCIENCE1"
+                                          :Topic "HUMAN DIMENSIONS1"
+                                          :Term "ENVIRONMENTAL IMPACTS1"
+                                          :VariableLevel1 "HEAVY METALS CONCENTRATION1"}
+                                         {:Category "EARTH SCIENCE2"
+                                          :Topic "HUMAN DIMENSIONS2"
+                                          :Term "ENVIRONMENTAL IMPACTS2"
+                                          :VariableLevel1 "HEAVY METALS CONCENTRATION2"}]}]
+       (side/eval-form `(ingest-config/set-bulk-update-enabled! false))
```

Comment:
It doesn't make sense to repeat these tests again. We should only be testing that multiple values can be provided (the only verification you should do for this test is lines 225-236 and get rid of everything else).

Reply:
Done.